tests/profile.yml

```diff
@@ -0,0 +1,161 @@
+version: 2.60.3-1.1
+jenkins_home: /var/lib/jenkins
+ports:
+  - 8080
+jenkins_url: http://localhost:8080
+jenkins_settings:
+  - key: "hudson/useSecurity"
+    value:
+      - 'true'
+  - key: "hudson/securityRealm/@class"
+    value:
+      - 'hudson.security.HudsonPrivateSecurityRealm'
+  - key: hudson/slaveAgentPort
+    value:
+      - '50000'
+
+jenkins_plugins:
```

Comment:
This list will change for every different version of Jenkins. Are we sure we want to hard-code it into the test instead of base it off the version of Jenkins in an external file, the way we do with the main plugin list? This could get to be a very noisy file, otherwise.